src/renderers/dom/shared/hooks/ReactDOMUnknownPropertyHook.js

```diff
@@ -67,6 +67,11 @@ if (__DEV__) {
     if (EventPluginRegistry.registrationNameModules.hasOwnProperty(name)) {
       return true;
     }
+    if (EventPluginRegistry.plugins.length === 0 && name.indexOf('on') === 0) {
```

Comment:
Regex shouldn't be that much slower in practice.

![image](https://user-images.githubusercontent.com/6820/28588953-02f7ed8c-7131-11e7-8859-27ca9d84fd3b.png)